sandbox-tools/merge-repos/src/syncReposToStaging.ts

```diff
@@ -389,33 +360,45 @@ async function resolveConflictsToTheirs(git: SimpleGit, commitDetails: ICommitDe
                     "--theirs",
                     conflicted
                 ]);
-                await git.add(conflicted);
+
+                await git.raw([
+                    "add",
+                    "-f",
+                    conflicted]);
             } else if (fileStatus.working_dir === "M") {
                 commitMessage = logAppendMessage(commitMessage, fileStatus, "Added in theirs, modified in ours => checkout theirs");
                 await git.checkout([
                     "--theirs",
                     conflicted
                 ]);
-                await git.add(conflicted);
+
+                await git.raw([
+                    "add",
+                    "-f",
+                    conflicted]);
             } else if (fileStatus.working_dir === "D") {
                 commitMessage = logAppendMessage(commitMessage, fileStatus, "Added in theirs, deleted in ours => checkout theirs");
                 await git.checkout([
                     "--theirs",
                     conflicted
                 ]);
-                await git.add(conflicted);
+
+                await git.raw([
+                    "add",
+                    "-f",
+                    conflicted]);
             } else if (fileStatus.working_dir === "U") {
-                commitMessage = logAppendMessage(commitMessage, fileStatus, "Added in ours => try to checkout theirs");
-                try {
-                    await git.checkout([
-                        "--theirs",
-                        conflicted
-                    ]);
-                    await git.add(conflicted);
-                } catch (e) {
-                    commitMessage = logAppendMessage(commitMessage, fileStatus, "!!! Unable to checkout theirs so assuming it should be deleted");
-                    await git.rm(conflicted);
-                }
+                commitMessage = logAppendMessage(commitMessage, fileStatus, "Added in ours => checkout ours");
```

Comment:
This was another automatic merge logic issue for cases where `git merge` matched files from `auto-merge/contrib` with `auto-merge/js` files and actually "matched" the JS version.
The previous logic was to delete the file here, the new logic is to keep the current file and let the new "validation" step remove the file if it "really" needs to be removed (after validting against the real source structure)